### PR TITLE
Rename server: salt/bundle / proxy

### PR DESCRIPTION
**Pull request by keichwa**
https://github.com/SUSE/spacewalk/issues/22831

*Before you submit this PR make sure you have added an entry to **.changelog** file.*

# Description

Short summary of why you created this PR (if you added documentation, please add any relevant diagram).

# Target branches

* Which documentation version does this PR apply to?
* Does this content need to be backported? If this is the case, create an issue for tracking it and add the link to this PR.
* Whenever possible, cross-reference each backport PR here, so that all backports can be easily accessed from the description.

- [ ] master
- [ ] manager-4.3

# Links
- This PR tracks issue #<insert spacewalk issue, if any>
- Related development PR #<insert PR link, if any>

### CODE REVIEW

**Review by 0rnela — APPROVED** [review]

File: modules/administration/pages/troubleshooting/tshoot-hostname-rename.adoc [hunk 1]
```diff
@@ -47,25 +47,37 @@ The only mandatory parameter for the script is the newly configured IP address o
...
 master: <new_hostname>
 ----
-+
-. Once this is changed, restart the [literal]``salt-minion` process:
+
+. Restart the [literal]``salt-minion`` process:
 +
 ----
 systemctl restart salt-minion
 ----
```
Comment (nodeg) [inline modules/administration/pages/troubleshooting/tshoot-hostname-rename.adoc:64]: I think this also needs to be adjusted for the Salt bundle (`vens-salt-minion`) vs. Salt (`salt-minion`).

**Comment by keichwa** [comment]
Dominik Gedon ***@***.***> writes:

>  ----
>  systemctl restart salt-minion
>  ----
>
> I think this also needs to be adjusted for the Salt bundle
> (`vens-salt-minion`) vs. Salt (`salt-minion`).

Thanks for spotting!

-- 
Karl Eichwalder

File: modules/administration/pages/troubleshooting/tshoot-hostname-rename.adoc [hunk 2]
```diff
@@ -47,25 +47,44 @@ The only mandatory parameter for the script is the newly configured IP address o
...
+
+. *Skip this step if the clients are managed via a {productname} proxy.*
+  Re-configure the clients directly managed to make them aware of the new hostname and IP address.
+  In the {salt} client configuration file, you must specify the name of the new {salt} master ({productname} Server) (the filename is [path]``/etc/venv-salt-bundle/minion``or, if you do not use the {salt} bundle, [path]``/etc/salt/minion``):
 +
 ----
 master: <new_hostname>
 ----
+
+. Restart the [systemitem]``venv-salt-minion`` service (if using the bundle):
```
Comment (nodeg) [inline modules/administration/pages/troubleshooting/tshoot-hostname-rename.adoc:60]:
```suggestion
. Restart the [systemitem]``venv-salt-minion`` service (if using the {salt} bundle):
```

File: modules/administration/pages/troubleshooting/tshoot-hostname-rename.adoc [hunk 3]
```diff
@@ -47,25 +47,44 @@ The only mandatory parameter for the script is the newly configured IP address o
...
++
+----
+systemctl restart venv-salt-minion
+----
+
 +
-. Once this is changed, restart the [literal]``salt-minion` process:
+Or  the [literal]``salt-minion``  service (if not using the bundle):
```
Comment (nodeg) [inline modules/administration/pages/troubleshooting/tshoot-hostname-rename.adoc:67]:
```suggestion
Or  the [literal]``salt-minion``  service (if not using the {salt} bundle):
```

**Review by nodeg — CHANGES_REQUESTED** [review]

File: modules/administration/pages/troubleshooting/tshoot-hostname-rename.adoc (hunk 1 above)
Comment (keichwa) [reply modules/administration/pages/troubleshooting/tshoot-hostname-rename.adoc]: thanks for your patience!

**Review by nodeg — APPROVED** [review]

**Pull request merged**